src/client.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+
+export default class SleuthClient {
+    constructor(url) {
+        this.url = url;
+    }
+
+    /**
+     * Returns a Response object in JSON format.
+     * @param {Response} response The response to our request that the `fetch`
+     * API returns.
+     */
+    static _json(response) {
```

Comment:
where does this get used?

Reply:
Oh man yea this is supposed to be called from _get(), but clearly it's not because _get() doesn't call it properly. This reveals an even bigger bug... the page seems to be doing a full reload when we submit the form. I'll fix it.

Reply:
@bfbachmann also just a side note but it appears I don't have write access to this repo (not collaborator)

Reply:
Fixed now.